lib/request.js

```diff
@@ -225,7 +225,7 @@ module.exports = {
    */
 
   get host() {
-    const proxy = this.app.proxy;
+    const { proxy } = this.app;
```

Comment:
I still think `const proxy = this.app.proxy;` is better on there is only one property read from app.